tcp/src/tcp_async.cc

```diff
@@ -163,6 +163,9 @@ void tcp_async::register_socket(asio::ip::tcp::socket& socket) {
   }
 }
 void tcp_async::unregister_socket(asio::ip::tcp::socket& socket) {
+  logging::error(logging::high)
+    << "tcp_async: unregister_socket";
+
```

Comment:
i think this is a leftover, this could appen quite  often. If you want to keep I think that  a logging::info using high or medium will be more appropriate..